src/components/Grid/Grid.tsx

```diff
@@ -853,6 +857,20 @@ export const Grid: FC<GridProps> = ({
     [arrowItems, renderArrow],
   );
 
+  const childrenArrowIndicators = useMemo(
+    () =>
+      arrowIndicators.map(position => {
+        return (
+          <ArrowIndicator
+            position={position}
```

Comment:
A `key` property with a unique value should be added to all components made in a loop.

```suggestion
            key={`${position.x}-${position.y}`}
            position={position}
```